acceptance/setup/packages/010_Install.rb

```diff
@@ -32,15 +41,20 @@
 # Install puppet on all hosts including master
 install_packages_on(hosts, PACKAGES)
 
+step 'Install build dependencies on master'
+
+MASTER_PACKAGES = {
+  :redhat => [
+    'git',
+    'java-1.8.0-openjdk-devel',
+  ],
+}
+install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+
 step 'Install MSIs on any Windows agents'
 agents.each do |agent|
   if agent.platform.start_with?('windows')
     logger.info "Installing Puppet agent msi #{sha} on #{agent}"
     install_puppet_agent_dev_repo_on(agent, :version => sha)
-    logger.info 'Prevent Puppet Service from Running'
```

Comment:
Ah. I copied the code from another project; on the assumption that this was the code that was needed to stop Puppet automatically running. Thanks for explaining